### DISCUSSION TITLE
Using same account to scan.

### DISCUSSION THREAD

**Discussion by Neeeap** [ThioJoe/YT-Spammer-Purge#866] [Q&A] [answered]
I'm wondering if I make a 2nd client secret file and scan comments with that file using the same account, will the comments that I reported with the 1st file using that account still be sent to review to see if it violates the guidelines?

**Comment by Firecul** [comment, marked as answer]
Yes, every time you scan it will report everything you tell it to, it wouldn't exclude the ones you have already reported.
